modules/aerodyn/src/AeroDyn.f90

```diff
@@ -4374,46 +4382,62 @@ SUBROUTINE TFin_CalcOutput(p, p_AD, u, m, y, ErrStat, ErrMsg )
 
    if (p%TFin%TFinIndMod==TFinIndMod_none) then
       V_ind = 0.0_ReKi
+
    elseif(p%TFin%TFinIndMod==TFinIndMod_rotavg) then
       ! TODO TODO
       print*,'TODO TailFin: compute rotor average induced velocity'
       V_ind = 0.0_ReKi 
+
    else
       STOP ! Will never happen
```

Comment:
Let's just respect the openfast convention here and instead of STOP do:
setErrStat(ErrID_Fatal, 'TailFin model unsupported', ErrStat, ErrMsg, 'TFin_CalcOutput')
return